python/tests/test_search_disk_index.py

```diff
@@ -1,105 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT license.
```

Comment:
can we move these files that you are deleting to ROOT/python/apps/  or ROOT/apps/ folder instead with suitable modifications.
These would be good examples for people to use our python interface. We would want to document such a workflow in workflows/python_interface.md as well.

Reply:
originally I had a folder named "examples" inside of python but it was using the native interface, not the python wrapper.  I can certainly come up with some good examples though!

Reply:
#303 will make sure we get those examples and documentation added prior to actual release